xata/utils.go

```diff
@@ -161,11 +161,11 @@ func loadConfig(fieName string) (config, error) {
 	return cfg, nil
 }
 
-// getBranchName retrieves the branch name.
-// If not found, falls back to defaultBranchName
-func getBranchName() string {
-	if branchName, found := os.LookupEnv(branchNameEnvVar); found {
-		return branchName
+// Get value from env var with fallback to godotenv
+// return default value if not found
+func getEnvVar(name string, defaultValue string) string {
+	if val, found := os.LookupEnv(name); found {
+		return val
```

Comment:
I noticed that if the .env file doesn't exist we return an empty string instead of the default value, is that correct?

Reply:
Yes, that is correct. It's fine for now, as we only fetch an env var to bootstrap the connection params and if an empty string is encountered, continues to probe the `.xatarc` file.